### DISCUSSION TITLE
Sushi

### DISCUSSION THREAD

**Discussion by Boom-H** [DiscordDungeons/Bugs#995] [Ideas]
Is sushi a good idea? I envisioned it like a tide pod, sells for a lot, but it heals instead of depleting, and you would use fish like tuna to craft it along with a recipe for rice (wheat and coal?) and a chance to get seaweed (30 or something? it's common i would think). It would make a use for fish, which don't really have a use, except for selling.

**Comment by Sven65** [comment]
Might be useful to implement in the cooking system from #1017
